lib/vagrant-service-manager/command.rb

```diff
@@ -34,7 +34,7 @@ def execute
           when 'docker'
```

Comment:
@bexelbie I have way to refactor `execute` method content. It will reduce to only one or two lines instead of its never ending content.
On my way to it :smile: